content/perl/workingWithVlans.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Working with Vlans"
+description: "A collection of CLI Examples on how to interact with Vlans."
+date: "2022-04-21"
+classes: 
+    - "SoftLayer_Product_Item_Price"
+    - "SoftLayer_Product_Package"
+    - "SoftLayer_Product_Order"
+    - "SoftLayer_Container_Product_Order_Network_Vlan"
```

Comment:
This tag it's not working
![image](https://user-images.githubusercontent.com/43519769/164731308-60cc47f2-36bc-4d8c-90b6-8be20f3f8d55.png)

Reply:
Classes were updated